modules/openxr/SCsub

```diff
@@ -18,10 +18,12 @@ env_openxr.Prepend(
         thirdparty_dir + "/src",
         thirdparty_dir + "/src/common",
         thirdparty_dir + "/src/external/jsoncpp/include",
-        thirdparty_dir + "/src/loader",
     ]
 )
 
+if env["platform"] == "android":
```

Comment:
I would do the same here as with the other PR, just move the whole block forward so our OpenXR includes within the module source also resolve properly.

Reply:
Done.